src/System.Net.Requests/tests/FtpWebRequestTest.cs

```diff
@@ -70,7 +70,7 @@ public void GetResponse_ServerNameNotInDns_ThrowsWebException()
         public void GetResponse_ConnectFailure_ThrowsWebException()
         {
             // This is an invalid IP address, so we should fail to connect.
-            string serverUrl = "ftp://192.0.2.1/";
+            string serverUrl = "ftp://127.0.0.1/";
```

Comment:
I think this solution is brittle because many devs might have an FTP server on their dev box.

Usually, for testing failure scenarios, we prefer to use what we call "pure unit tests".  Those are tests that do not depend on the real network at all.  They are tests that are built into test assemblies that bring in product sources to build with and only include those APIs and internal files necessary to compile.  We then would mock the network and generate any kind of failure condition we want.

See WinHttpHandler unit tests for example which mock the native WinHTTP layer.

So, I think the best solution here is to use an approach like that which will be 100% stable on any machine and any network configruation.

cc: @karelz @CIPop